uint8 representation
uint8 REPRESENTATION_BOX = 0

string id
# Note, if null string then will be automatically generated

geometry_msgs/Vector3 size
geometry_msgs/Pose pose
